corehq/apps/locations/util.py

```diff
@@ -118,7 +118,7 @@ def get_location_data_model(domain):
 class LocationExporter(object):
 
     def __init__(self, domain, include_consumption=False, root_location_id=None,
-                 headers_only=False, async_task=None):
+                 headers_only=False, async_task=None, **kwargs):
```

Comment:
It would be cool if `LocationExporter` changes were covered by tests.

Reply:
Added some tests [here](https://github.com/dimagi/commcare-hq/blob/c55a556f15da2717cca7648c4debc1f64d6dbdce/corehq/apps/locations/tests/test_export.py#L105).